src/test/java/com/redhat/exhort/integration/backend/BackendUtilsTest.java

```diff
@@ -34,10 +34,16 @@
 class BackendUtilsTest {
 
   private BackendUtils backendUtils;
```

Comment:
Do you really need both? I think using the fakeTimer only is enough for the tests.

Reply:
@ruromero I Also need real timestamp, for all other cases that time needs to be "natural" , especially for the mass concurrency test.

Reply:
I didn't notice it's a fixed date, You're right.